dynet/io.cc

```diff
@@ -84,37 +86,30 @@ void TextFileSaver::save(const LookupParameter & param,
 
 void TextFileSaver::save(const ParameterStorage & p,
                          const string & key) {
-  std::ostringstream buffer;
-  buffer.precision(FLOAT32_PRECISION);
-  buffer << dynet::as_vector(p.values) << endl;
+  datastream << "#Parameter# " << (key.size() > 0 ? key : p.name) << ' ' << p.dim << ' ';
+  size_t strsize = p.dim.size() * (FLOAT32_PRECISION + 8) + 1;
```

Comment:
According to [this](http://www.cplusplus.com/reference/ios/scientific/), it says there is three exponential digits, but on my machine, there is only two.